src/assets/js/main-ex.js

```diff
@@ -1,5 +1,42 @@
 $(function(){
 
+    // Retrieve category by selected get params
```

Comment:
Alot of unnecessary duplication in this js code. Clean this up please.

I don't think we should use `console.log` in poduction either